src/views/ApartmentApp/index.js

```diff
@@ -39,34 +54,24 @@ export default class ApartApp extends Component {
   };
 
   async addApplicant(username) {
-    try {
-      // Get the profile of the selected user
-      let applicantProfile = await user.getProfileInfo(username);
-      // Check if the selected user is a student
-      if (String(applicantProfile.PersonType).includes('stu')) {
-        let applicants = this.state.applicants; // make a separate copy of the array
-        // Check if new applicant is already in list
-        if (!applicants.some(applicant => applicant.AD_Username === username)) {
-          // Add the profile object to the list of applicants
-          applicants.push(applicantProfile);
-          this.setState({ applicants });
-        } else {
-          // Display an error if the selected user is already in the list
-          let newErrorMessage = 'Error: ' + applicantProfile.fullName + ' is already in the list';
-          this.setState({ errorDialogOpen: true });
-          console.log(newErrorMessage);
-          alert(newErrorMessage);
-        }
-      } else {
-        // Display an error if the selected user is not a student
-        let newErrorMessage = 'Error: ' + applicantProfile.fullName + ' is not a student';
-        this.setState({ errorDialogOpen: true });
-        console.log(newErrorMessage);
-        alert(newErrorMessage);
-      }
-    } catch (error) {
-      // Do Nothing
-      alert('Something went wrong while trying to get the profile of ' + username);
+    let applicants = this.state.applicants; // make a separate copy of the array
+    // Get the profile of the selected user
+    let applicantProfile = await user.getProfileInfo(username);
+    // Check if the selected user is a student
+    if (!String(applicantProfile.PersonType).includes('stu')) {
+      // Display an error if the selected user is not a student
+      let newErrorText = applicantProfile.fullName + ' is not a student';
+      this.setState({ errorDialogOpen: true, errorDialogText: newErrorText });
+      console.log('Error: ' + newErrorText);
```

Comment:
Is this console log leftover from testing?

Reply:
Yes, I guess there is no need for it anymore, now that the stylized dialog has been added